helpers/github.ts

```diff
@@ -62,6 +62,22 @@ export async function forceCloseMissingIssues(devpoolIssues: GitHubIssue[], proj
   }
 }
 
+/**
+ * Stops forks from spamming real Ubiquity issues with links to their forks
+ * @returns true if the authenticated user is Ubiquity
+ */
+export async function isUbq() {
+  try {
+    const {
+      data: { login },
+    } = await octokit.users.getAuthenticated();
+    return login && login === "ubiquity" ? true : false;
```

Comment:
Perhaps it would be more robust to use an organization ID? 

```json
{
  "login": "ubiquity",
  "id": 76412717,
  "node_id": "MDEyOk9yZ2FuaXphdGlvbjc2NDEyNzE3",
  "url": "https://api.github.com/orgs/ubiquity",
  "repos_url": "https://api.github.com/orgs/ubiquity/repos",
  "events_url": "https://api.github.com/orgs/ubiquity/events",
  "hooks_url": "https://api.github.com/orgs/ubiquity/hooks",
  "issues_url": "https://api.github.com/orgs/ubiquity/issues",
  "members_url": "https://api.github.com/orgs/ubiquity/members{/member}",
  "public_members_url": "https://api.github.com/orgs/ubiquity/public_members{/member}",
  "avatar_url": "https://avatars.githubusercontent.com/u/76412717?v=4",
  "description": "The Metaverse Bank.",
  "name": "Ubiquity DAO",
  "company": null,
  "blog": "https://ubiquitydao.xyz",
  "location": null,
  "email": "github.home@ubiquitydao.xyz",
  "twitter_username": "UbiquityDAO",
  "is_verified": true,
  "has_organization_projects": true,
  "has_repository_projects": true,
  "public_repos": 50,
  "public_gists": 0,
  "followers": 78,
  "following": 0,
  "html_url": "https://github.com/ubiquity",
  "created_at": "2020-12-21T00:21:13Z",
  "updated_at": "2024-02-15T01:23:04Z",
  "archived_at": null,
  "type": "Organization"
}
```

Reply:
`octokit.users.getAuthenticated();` seems like a convenient method so lets stick with it.